ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+  # Main help command
```

Comment:
Can you change your indentation throughout this script from 2 spaces to 4? I don't think we document this convention anywhere in our internal Nextstrain docs, but future editors of this code will likely get thrown by the unexpected indentation here.

Reply:
Gah, my convention in my other scripts. I didn't even notice I did that. Will swap to 4 spaces